src/tests/_suite.ts

```diff
@@ -18,7 +18,7 @@ describe('azure pipeline task tests', function() {
       'uri',
       true,
       true,
-      'abcs',
+
```

Comment:
Does this need to be run through prettier? A bit weird to have a random line like this.

Reply:
Prettier? What's that. This repo is only our second example of TS :grin:

Reply:
Thank for the suggestion.

Much prettier now. :lipstick: